#167101210: Fix developer card component

johnkegz opened the pull request:
#### What does this PR do?
This PR fixes the `developerCard` Component

#### Description of Task to be completed?
#####  Current behaviour
- There is only one developer-card component ever in the cards view, which then renders each automation as a div within itself.
- After viewing the details of automation, subsequent automation details get mixed up with previously viewed automation details
##### Expected
- Each (visual) card should be a developer-card component
- The details in the modal should match the details of the automation being viewed
#### How should this be manually tested?
##### First 
- Run `yarn` in the frontend to install new dependencies
- Log in to the application and click on the card view
- Inspect and go to react developer tools
- Search for `DeveloperCard` component and you will realize that there are individual `developerCard` components for each card.
##### second
- Click on info icon for different cards and you will realize that details in the modal match the details of the automation being viewed.
#### Any background context you want to provide?
- The PR works with this [backend PR](https://github.com/andela/bp-esa-backend/pull/118)

#### What are the relevant pivotal tracker stories?
[167101210](https://www.pivotaltracker.com/story/show/167101210)
#### Postman Documentation
- N/A

#### Screenshots (If applicable)
<img width="1440" alt="Screen Shot 2019-07-16 at 12 11 26" src="https://user-images.githubusercontent.com/29975767/61281788-e640a100-a7c2-11e9-97b7-b1290b0294ba.png">

katunold reviewed (commented):
This Looks good, but you still have failing tests to work on. Otherwise well done

johnkegz committed:
bg(frontend)fix developer card component

 - each (visual) card should be a developer-card component
 - details in the modal should match the details of the automation being viewed

[Fixes #167101210]

ssewilliam commented on src/components/developerCards/index.jsx:
```diff
@@ -71,67 +70,61 @@ class DeveloperCard extends Component {
+    const name = card.fellowName;
+    const newName = _.split(name, ',');
+    const firstInitials = newName[0].charAt(0);
+    const secondInitials = newName[1].charAt(1);
```
@johnkegz  This change is likely to return the bug that was solved [here](https://www.pivotaltracker.com/story/show/167191688) and fixed through this [PR](https://github.com/andela/bp-esa-frontend/pull/101)

johnkegz replied:
@ssewilliam, the application is not returning the bug you have referenced in the `story` and `PR` above
